addons/web/static/src/core/notifications/notification.scss

```diff
@@ -1,48 +1,23 @@
-$notification-width: 320px;
-$notification-text-color: #212529;
-$notification-title-text-color: #6c757d;
-
 // Colors of the notification are not managed by this file.
 // It is handled by the other classes defining a background. (like danger, warning, etc.)
 // The notification just add overlays.
 
 .o_notification_manager {
-  @include o-position-absolute(3.5rem, 0);
-  position: fixed;
+  @include o-position-absolute(4.25rem, 0);
   z-index: ($zindex-modal + $zindex-popover) / 2;
-  width: $notification-width;
-  margin-right: 0.5rem;
-  max-width: 100%;
+  width: $o-notification-width;
+  right: 0.5rem;
```

Comment:
remove and correctly set the `o-position-absolute` mixin `right`value (using default spacers)